support-frontend/assets/pages/supporter-plus-landing/twoStepPages/secondStepCheckout.tsx

```diff
@@ -144,16 +143,6 @@ export function SupporterPlusCheckout({
 	const showCoverTransactionCost =
 		abParticipations.coverTransactionCost === 'variant';
 
-	/** Promotions on the checkout are for SupporterPlus only for now */
-	const promotion = isSupporterPlus
-		? useContributionsSelector((state) =>
-				getPromotion(
-					state.page.checkoutForm.product.productPrices,
-					countryId,
-					state.page.checkoutForm.product.billingPeriod,
-				),
-		  )
-		: undefined;
```

Comment:
This has been removed as this checkout is no longer used for anything but one time payments.